index.html

```diff
@@ -0,0 +1,227 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>portfolio</title>
+    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/css/bootstrap.min.css">
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="logo-nav">
+            <div class="logo">
+            <h1>My Logo</h1>
+            </div>
+            <div class="nav-bar">
+                <ul>
+                    <li><a href="#portfolio-1">portfolio</a></li>
+                    <li><a href="#about-1">about</a></li>
+                    <li><a href="#contact-1">contact</a></li>
+                </ul>
+            </div>
+        </div>
+        <div class="container-1">
+            <div class="discription">
+                <h1>I'm Girma Glad to see you!</h1>
+                <p>I'm a software developer! I can help you build a product, feature or website look through some of my work and experience! if you like what you  see and have a project you need coded, don't hestiate to contact me</p>
+                <div class="let-s">
+                    <h1>LET'S CONNECT</h1>
+                    <div class="vector">
+                        <ul>
+                            <li><a href="https://github.com/girma336" class="twitter"></a></li>
+                            <li><a href="#" class="linkedin"></a></li>
+                            <li><a href="#" class="medium"></a></li>
+                            <li><a href="#" class="github"></a></li>
+                            <li><a href="#" class="hand"></a></li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <section class="main-part" id="portfolio-1">
+        <article>
+            <div class="back-image-file all-image-pro"></div>
+            <div class="back-description">
+                <h1>Tonic</h1>
+                <div>
+                    <ul class="page-list">
+                        <p>CANOPY</p>
+                        <li>Back End developer</li>
+                        <li>2015</li>
+                    </ul>
+                </div>
+                <div>
+                    <p class="uniqe-para">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                </div>
+                <div class="nav-source">
+                    <ul>
+                        <li>html</li>
+                        <li>JavaScript</li>
+                        <li>css</li>
+                    </ul>
+                </div>
+                <div class="button-m">
+                    <button type="button">See Project</button>
+                </div>
```

Comment:
### Your Button
![image](https://user-images.githubusercontent.com/93039359/194259703-5f3dffa1-eb29-42ce-8977-2d0460e68e41.png)
### Figma Button
![image](https://user-images.githubusercontent.com/93039359/194259779-0de68dbc-622c-463b-87ff-6c665a81a333.png)

- [ ] Your button styling is not very much similar to the Figma template. Kindly follow the Figma template to make your design look professional which whill attract the recruiters :+1:

Reply:
Fixed.